src/containers/my-courses/add-course-with-input/AddCourseWithInput.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC, ReactElement } from 'react'
+import { Link } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import AddIcon from '@mui/icons-material/Add'
+import SearchIcon from '@mui/icons-material/Search'
+import Box from '@mui/material/Box'
+
+import AppButton from '~/components/app-button/AppButton'
+import InputWithIcon from '~/components/input-with-icon/InputWithIcon'
+
+import { authRoutes } from '~/router/constants/authRoutes'
+import CoursesFilterBar from '~/containers/find-course/courses-filter-bar/CoursesFilterBar'
+import { styles } from '~/containers/my-courses/add-course-with-input/AddCourseWithInput.styles'
+
+interface AddCourseWithInputProps {
+  button?: ReactElement
```

Comment:
Remove this interface, for now, it is redundant

Reply:
I saw this but forgot to change it, thanks